core/src/main/scala/org/apache/spark/ui/jobs/StageTable.scala

```diff
@@ -194,7 +207,7 @@ private[ui] class FailedStageTable(
 
   override protected def stageRow(s: StageInfo): Seq[Node] = {
     val basicColumns = super.stageRow(s)
-    val failureReason = s.failureReason.getOrElse("")
+    val failureReason = s.failureReason.getOrElse("-")
```

Comment:
Super nit, but does this need to change? It seems like changing it to "-" is inconsistent with the way we handle other missing information (e.g., when there's no shuffle read data, we leave it blank, instead of having "-")